README.md

```diff
@@ -87,8 +87,7 @@ Add the following code snippet, replacing the placeholders with your app’s inf
 <key>UberCallbackURI</key>
 <string>[redirect URL]</string>
 ```
-
-Make sure UberCallbackURI matches one, which is set on developer dashboard (use localhost for testing).
+Make sure the value for UberCallbackURI exactly matches one for the Redirect URLs you have set on your developer dashboard. (You can use `localhost` for testing.)
```

Comment:
Small typo still, we want "of" instead of "for" in "...exactly matches one of the Redirect URLs..."